configure.ml

```diff
@@ -329,6 +330,8 @@ let args_options = Arg.align [
     " Add profiling information in the Coq executables";
   "-annotate", Arg.Set Prefs.annotate,
     " Dumps ml annotation files while compiling Coq";
+  "-enable-vm", arg_bool Prefs.enable_vm,
```

Comment:
Maybe use a flag name similar to `-native-compiler`, e.g. `-vm-compiler`?

Reply:
As you prefer, any other thoughts on the naming before I update?

Reply:
`-bytecode-compiler`?